testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/cli/admin/KcAdmTest.java

```diff
@@ -684,4 +684,20 @@ public void testEnvPasswordWithRegularCommand() {
         assertExitCodeAndStreamSizes(exec, 0, 1, 0);
     }
 
+    @Test
+    public void testStatusOptionForConfigCred() {
+        KcAdmExec exec = execute("config credentials --status");
+        assertExitCodeAndStreamSizes(exec, 0, 1, 0);
+        Assert.assertEquals("You are not logged in\n", exec.stdoutString());
```

Comment:
@keshavprashantdeshpande these tests have a user logged in by default, so this first assertion is failing.

Reply:
@shawkins I have removed the assert and checks seem to work again.  Can you please review the changes?